ui-tests/src/test/java/io/syndesis/qe/pages/integrations/fragments/IntegrationsList.java

```diff
@@ -0,0 +1,66 @@
+package io.syndesis.qe.pages.integrations.fragments;
+
+import static com.codeborne.selenide.Condition.visible;
+import static com.codeborne.selenide.Selenide.$;
+
+import org.openqa.selenium.By;
+
+import com.codeborne.selenide.SelenideElement;
+
+import io.syndesis.qe.fragments.common.list.RowList;
+import io.syndesis.qe.fragments.common.list.actions.ListAction;
+import io.syndesis.qe.fragments.common.menu.KebabMenu;
+import io.syndesis.qe.pages.ModalDialogPage;
+
+public class IntegrationsList extends RowList {
+
+    public IntegrationsList(By rootElement) {
+        super(rootElement);
+    }
+
+    private static final class Element {
+        public static final By ITEM = By.xpath("//*[contains(@class,'list-pf-item')]");
+        public static final By STATUS = By.xpath("//div[contains(@class,'status')]/span");
+        public static final By DESCRIPTION = By.className("description");
+    }
+
+    private static final class Button {
+        public static final By KEBAB = By.cssSelector("button.dropdown-toggle");
+    }
+
+    @Override
+    public boolean invokeActionOnItem(String title, ListAction action) {
+        if(super.invokeActionOnItem(title, action)) {
```

Comment:
Maybe I'd prefer the opposite behaviour, let the action be. Let the override method go first then in invoke super method in `default: return super.invokeActionOnItem(title, action);` Since we probably want to invoke `Delete` anyway as only implemented method of this child class.

Reply:
good idea, fixed